server/src/domain/search/search.service.ts

```diff
@@ -211,4 +211,10 @@ export class SearchService {
       }
     }
   }
+
+  async getAssetsByCity(auth: AuthDto): Promise<AssetResponseDto[]> {
+    const userIds = await this.getUserIdsToSearch(auth);
+    const assets = await this.searchRepository.getAssetsByCity(userIds);
+    return assets.map((asset) => mapAsset(asset));
+  }
```

Comment:
Can we move that method up? I think having all private methods at the bottom generally is preferred...?

Reply:
Agreed. I moved it and the method above it above the private methods.